packages/sdk/src/schema/schema.ts

```
@@ -29,6 +28,7 @@ export type ModelDef = {
         | Record<string, Pick<FieldDef, 'type'>>
     >;
     idFields: string[];
+    // eslint-disable-next-line @typescript-eslint/ban-types
     computedFields?: Record<string, Function>;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)